nodes/index.adoc

```diff
@@ -129,3 +129,53 @@ As an administrator, You can perform various tasks on a Linux container, such as
 {product-title} provides specialized containers called xref:../nodes/containers/nodes-containers-init.adoc#nodes-containers-init[Init containers]. Init containers run before application containers and can contain utilities or setup scripts not present in an application image. You can use an Init container to perform tasks before the rest of a pod is deployed.
 
 Apart from performing specific tasks on nodes, pods, and containers, you can work with the overall {product-title} cluster to keep the cluster efficient and the application pods highly available.
+
+[id="commonterms-node"]
+== Common terms
+
+This glossary defines common terms that are used in the _node_ content. These terms help you understand _nodes_ effectively.
+
+Container::
+It is a lightweight and executable image that comprises software and all its dependencies. Containers virtualize the operating system, as a result, you can run containers anywhere, from a data center to a public or private cloud to even a developer's laptop.
+
+Daemon set::
+It ensures that a replica of the pod runs on eligible nodes in an {product-title} cluster.
```

Comment:
```suggestion
Ensures that a replica of the pod runs on eligible nodes in an {product-title} cluster.
```
Just for clarity and to be more direct, I think it would help readers to avoid using terms like "it" and "it is" when introducing a definition.